smoke-test.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -e
+
+TARBALL_PREFIX=dotnet-sdk-
+VERSION_PREFIX=2.1
+OUTPUT_DIR=bin/x64/Release/
+DOTNET_TARBALL_REL=$(ls ${OUTPUT_DIR}dotnet-sdk-${VERSION_PREFIX}*)
+DOTNET_TARBALL=$(readlink -e ${DOTNET_TARBALL_REL})
+
+projectOutput=false
+keepProjects=false
+dotnetDir=""
+workingDir="$(pwd)"
+includeWeb=false
+
+function usage {
+    echo ""
+    echo "usage:"
+    echo "  --dotnetDir         the directory from which to run dotnet"
+    echo "  --projectOutput     echo dotnet's output to console"
+    echo "  --keepProjects      keep projects after tests are complete"
+    echo "  --includeWeb        run tests for web projects"
+    echo "  --clean             remove the testing directory if it exists"
+    echo ""
+}
+
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
+        -?|-h|--help)
+            usage
+            exit 1
+            ;;
+        --dotnetdir)
+            shift
+            dotnetDir="$1"
+            ;;
+        --projectoutput)
+            projectOutput=true
+            ;;
+        --keepprojects)
+            keepProjects=true
+            ;;
+        --includeweb)
+            includeWeb=true
+            ;;
+        --clean)
```

Comment:
I think this could be the default, I've tried to run smoke tests and ran into the existing directory way more times than I ran it without wanting wanting to delete the previous version.

Reply:
Added an option to prompt for removal rather than an arg.